### PR TITLE
[DOC] Updated gitian documentation to reference docker process

### PR DESCRIPTION
**Pull request by codeofalltrades**
The documentation related to the Gitian build process was updated to reference this repo: https://github.com/Veil-Project/docker-based-gitian-builder

### CODE REVIEW

File: doc/gitian-building.md [hunk 1]
```diff
@@ -15,12 +12,19 @@ These results are compared and only if they match, the build is accepted and pro
...
+
+# Gitian building using a VM (Legacy)
+*Maintained for legacy purposes. The docker process above should be used for all new builds.*
+- *Setup instructions for a Gitian build of Veil using a VM or physical system.*
+- *You Should use Ubuntu 18.04.1 server for the VM host until further notice.*
 
 Table of Contents
 ------------------
-
 - [Preparing the Gitian builder host](#preparing-the-gitian-builder-host)
```
Comment (CaveSpectre11) [inline doc/gitian-building.md]: Check these links.  Looks like the first one jumps to the section but the other sections don't seem to point to the right places.

Comment (codeofalltrades) [reply doc/gitian-building.md]: Updated links. Removed duplicate file.

**Comment by crypt0-fusi0n** [comment]
@codeofalltrades Looks good, all links work except for these... which reference docs in the PR that don't exist... but do exist in master. Will these resolve itself after committing?

image

Also, is it fine to have @presstab 's email at the bottom there?

**Comment by codeofalltrades** [comment]
@crypt0-fusi0n looks like the links you pointed out are broken. I will fix and resubmit.

**Pull request merged**